### ISSUE TITLE
The validation set is the subset of training set?

### ISSUE THREAD

**Issue by ootts** [princeton-computational-imaging/neural-scene-graphs#5]
The code [here](https://github.com/princeton-computational-imaging/neural-scene-graphs/blob/main/data_loader/load_kitti.py#L764) shows that the training set contains all frames in the scene, and the validation (test) set is the part of the training set. Is this setting reasonable for a novel view synthesis task?

**Comment by ostjul** [comment]
The uploaded experiments are primarly showing the reconstruction task. For the novel view synthesis task you can use the "remove_frame" argument.
